opflexagent/vpp_plug_firewall.py

```diff
@@ -0,0 +1,88 @@
+import functools
+from os_vif import objects
+from oslo_log import log as logging
+
+from nova.network import model
+import nova.network.os_vif_util as os_vif_util
+
+LOG = logging.getLogger(__name__)
+
+
+class NoopFirewallDriver(object):
+    """Firewall driver which just provides No-op methods."""
```

Comment:
Is this the right class name and description?

Reply:
Mandeep's suggestion was to use the nova firewall driver to do monkey patching.  So this is a dummy firewall class. Let me know if you think this can be renamed better

Reply:
Just wondering whether this Firewall driver is actually a Noop. It doesn't look like it